src/Components/Main/TOC/Tab.js

```diff
@@ -0,0 +1,11 @@
+import Button from "./Button.js";
+import "./Tab.css";
+
+export default function Tab({ content, className }) {
+  return (
+    <>
+      <span className={className}>{content}</span>
+      <Button className={"button"}>3</Button>
```

Comment:
Was hat die 3 hier zu bedeuten?

Reply:
das hatte ich einfach hard coded, damit es wie in der Vorlage aussieht 🤷🏽
Im Normalfall würde ich es nie machen, aber es war so in der Aufgabe geschrieben, dass wir das so machen sollen. Alle andere Vorschläge von dir habe ich implementiert, vielen Dank 🙂
Das mit dem Datum ist auch hardcoded, ich habe den Konstruct zwar vorbereitet, aber noch nicht benutzt